liquibase-service/src/main/resources/db/rdb/tables/017-create_nrt_observation-003.sql

```diff
@@ -0,0 +1,249 @@
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD processing_decision_cd varchar(20);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD cd varchar(50);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'shared_ind' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD shared_ind char(1);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'status_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD status_cd char(1);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'cd_system_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD cd_system_cd varchar(50);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'cd_system_desc_txt' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD cd_system_desc_txt varchar(1000);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'ctrl_cd_user_defined_1' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD ctrl_cd_user_defined_1 varchar(20);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'alt_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD alt_cd varchar(50);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'alt_cd_desc_txt' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD alt_cd_desc_txt varchar(1000);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'alt_cd_system_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+            ALTER TABLE nrt_observation
+                ADD alt_cd_system_cd varchar(1000);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'alt_cd_system_desc_txt' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD alt_cd_system_desc_txt varchar(100);
+            END;
+
+        IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'method_cd' AND object_id = Object_ID(N'nrt_observation'))
+            BEGIN
+                ALTER TABLE nrt_observation
+                    ADD method_cd varchar(2000);
```

Comment:
Is the length 2000 for method_cd ?

Reply:
Source is observation's method_cd. That being said, Alabama dataset's max length is 68.

Reply:
Yes, same length in the original table in ODSE.